core/src/Cuda/Kokkos_Cuda_Graph_Impl.hpp

```diff
@@ -131,6 +132,7 @@ struct GraphImpl<Kokkos::Cuda> {
     kernel.set_cuda_graph_node_ptr(&cuda_node);
     kernel.execute();
     KOKKOS_ENSURES(bool(cuda_node));
+    // m_nodes.push_back(std::static_pointer_cast<node_details_t>(arg_node_ptr));
```

Comment:
I'm explicitly disabling the fix to see which CI job will fail.

Reply:
And my 2 cents is that the fix is needed for `Cuda` though the test on my machine was fine.

Reply:
Both backends needed the fix. Closing.